assets/react/v3/shared/components/fields/FormCategoriesInput.tsx

```diff
@@ -166,8 +166,13 @@ interface BranchProps {
   isLastChild: boolean;
 }
 
+// all children nested under this category
```

Comment:
```suggestion
// Calculate all children nested under this category
```
Maybe we don't need to add comment instead make the function name more descriptive

Reply:
change the name to `getTotalNestedChildrenCount`